src/utils/IndexKeyUtils.h

```diff
@@ -370,9 +382,8 @@ class IndexKeyUtils final {
                     break;
                 }
                 case Value::Type::STRING: {
-                    auto off = key.size() - vCount * sizeof(int32_t) - tailLen;
-                    len = *reinterpret_cast<const int32_t*>(key.c_str() + off);
-                    --vCount;
+                    len = (hasNullableCol && nullableBit.test(nullableColPosit))
```

Comment:
Why len is 1 when value is null?

Reply:
> Why len is 1 when value is null?

If the value type is string and it is null. only one byte is filled.